utils/http/response.py

```diff
@@ -4,3 +4,12 @@
 
 class HttpResponseAccepted(HttpResponse):
     status_code = 202
+
+
+class ResponseError(Exception):
```

Comment:
Might be better in `utils/http/errors.py` ?

Reply:
Done.